client/trino-jdbc/src/test/java/io/trino/jdbc/TestAsyncResultIterator.java

```diff
@@ -154,6 +154,12 @@ public boolean isFinished()
             return true;
         }
 
+        @Override
```

Comment:
Not needed, since you've added a default implementation.

Reply:
Removed.